paimon-core/src/main/java/org/apache/paimon/mergetree/compact/PartialUpdateMergeFunction.java

```diff
@@ -205,27 +205,8 @@ private Factory(Options options, RowType rowType) {
                     SequenceGenerator sequenceGen =
                             new SequenceGenerator(sequenceFieldName, rowType);
                     Arrays.stream(v.split(","))
-                            .map(
```

Comment:
We don't need to delete here, we can keep it as it is.

Reply:
Make sense, updated.